iocore/net/Connection.cc

```diff
@@ -390,6 +385,7 @@ Server::listen(bool non_blocking, const NetProcessor::AcceptOptions &opt)
     fd = NO_FD;
   }
 
-  Error("Could not bind or listen to port %d (error: %d)", ats_ip_port_host_order(&addr), res);
+  Fatal("Could not bind or listen to port %d, mptcp enabled: %d (error: %d) %s %d", ats_ip_port_host_order(&addr),
+        prot == IPPROTO_MPTCP, errno, strerror(errno), res);
```

Comment:
This makes sense. If you configured to listen on a port and can't, probably best to Fatal.

Reply:
Oh this is Fatal on master but maybe it should be consistent with 9.2.x. Thoughts?

Reply:
Ah, gotcha. Yeah, probably fine to pull it in as fatal for 9.2.x too while we're at it.